events.go

```diff
@@ -132,6 +132,18 @@ func convertLibcontainerStats(ls *libcontainer.Stats) *types.Stats {
 	s.CPU.Throttling.ThrottledPeriods = cg.CpuStats.ThrottlingData.ThrottledPeriods
 	s.CPU.Throttling.ThrottledTime = cg.CpuStats.ThrottlingData.ThrottledTime
 
+	s.CPUSet.CPUs = cg.CPUSetStats.CPUs
+	s.CPUSet.CPUExclusive = cg.CPUSetStats.CPUExclusive
+	s.CPUSet.Mems = cg.CPUSetStats.Mems
+	s.CPUSet.MemExclusive = cg.CPUSetStats.MemExclusive
+	s.CPUSet.MemHardwall = cg.CPUSetStats.MemHardwall
+	s.CPUSet.MemoryMigrate = cg.CPUSetStats.MemoryMigrate
+	s.CPUSet.MemoryPressure = cg.CPUSetStats.MemoryPressure
+	s.CPUSet.MemorySpreadPage = cg.CPUSetStats.MemorySpreadPage
+	s.CPUSet.MemorySpreadSlab = cg.CPUSetStats.MemorySpreadSlab
+	s.CPUSet.SchedLoadBalance = cg.CPUSetStats.SchedLoadBalance
+	s.CPUSet.SchedRelaxDomainLevel = cg.CPUSetStats.SchedRelaxDomainLevel
```

Comment:
Looks like these types are compatible, and this could be simplified to;

```suggestion
	s.CPUSet = types.CPUSet(cg.CPUSetStats)
```

Reply:
Tricky :) Thanks!